### PR TITLE
Add `Dispatcher::toCamelCase()` method

### PR DESCRIPTION
**Pull request by alexbusu**
Hello!

* Type: bug fix
* Link to issue: #12829 

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

Small description of change:

Added `Dispatcher::toCamelCase()` method to use with `::getHandlerClass()` and `::getActiveMethod()`.

Thanks

### CODE REVIEW

**Comment by niden** [comment]
@alexbusu you have a PHPCS issue. Check the result  of the build. Other than that this is solid

Thank you

File: tests/unit/Dispatcher/GetHandlerClassCest.php [hunk 1]
```diff
@@ -30,6 +31,66 @@ class GetHandlerClassCest
     public function dispatcherGetHandlerClass(UnitTester $I)
     {
         $I->wantToTest('Dispatcher - getHandlerClass()');
-        $I->skipTest('Need implementation');
+
```
Comment (niden) [inline tests/unit/Dispatcher/GetHandlerClassCest.php]: Can we break this test into 3 separate ones (prefix/suffix/namespace) or do some combination of those in one test using a dataProvider?

Have a look at this 

```php
use Codeception\Example;

/**
 * Blah blah description
 *
 * @dataProvider getExamples
 */
public function dispatcherGetHandlerClass(UnitTester $I, Example $example)
{
    $I->wantToTest('Dispatcher - getHandlerClass() - ' . $example[0]);
    $dispatcher = new Dispatcher();
    // test the handler name
    $dispatcher->setNamespaceName('');
    $dispatcher->setHandlerSuffix('');
    $dispatcher->setControllerName($example[1]);

    $actualHandler = $dispatcher->getHandlerClass();
    $I->assertEquals($example[2], $actualHandler);
}

private function getExamples(): array
{
    return [
        ['simple',     'hello',          'Hello'],
        ['dash',       'hello-phalcon',  'HelloPhalcon'],
        ['underscore', 'hello_phalcon',  'HelloPhalcon'],
        ['camelized',  'HelloPhalcon',   'HelloPhalcon'],
        ['with slash', 'Hello\\Phalcon', 'Hello\\Phalcon'],
    ];
}
```

Comment (niden) [reply tests/unit/Dispatcher/GetHandlerClassCest.php]: The `getExamples` can return more than those so we can use it to set the namespace and suffix as well. This way we have a test that we can extend with any new data/use case we get in the future.

**Review by niden — CHANGES_REQUESTED** [review]

**Comment by niden** [comment]
@alexbusu When you get a chance can you look at my comment regarding the test?

File: tests/unit/Dispatcher/GetHandlerClassCest.php (hunk 1 above)
Comment (alexbusu) [reply tests/unit/Dispatcher/GetHandlerClassCest.php]: @niden done.

**Comment by niden** [comment]
You are a superstar @alexbusu

**Review by niden — APPROVED** [review]

**Pull request merged**